### PR TITLE
Epic and banner should never render a %%variable%%

### PR DESCRIPTION
**Pull request by jlieb10**
## What does this change?
In very odd edge cases, variant copy is being built with invalid information so variables `%%this%%` appear in the copy. However, at the time the test runs, the valid information has become available and so the test runs with the invalid copy. This change will scan the variant copy after it has been generated and prevent a user from entering the test if any of the variant copy still has variables present.

Note - we think we know why this race condition involving geolocation occurs and will be making a fix in a separate PR. This PR just guards against this type of error ever occurring.

## Screenshots
Confirming it still works the same -

No country name available, default epic/banner:
Screenshot 2019-07-09 at 09 09 04

With country name:
Screenshot 2019-07-09 at 09 09 46

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js [hunk 1]
```diff
@@ -265,6 +268,12 @@ const makeEpicABTestVariant = (
         excludedSections: initVariant.excludedSections || [],
 
         canRun() {
+            const copyIsValid = () =>
+                !this.copy ||
```
Comment (tomrf1) [inline static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js]: the epic `copy` is optional in the model

**Review by tjsilver — APPROVED** [review]
Looks good to me.
